From 0ff57cb48d8d318ebd9a4f29001ee8ec9ffefabe Mon Sep 17 00:00:00 2001
From: rramakax <rohitx.kant@intel.com>
Date: Fri, 25 Sep 2020 09:10:20 +0530
Subject: [PATCH] [CIV] disable Chromium kernel readonly option for
 "/proc/self/mem".
 Fix For VTS failures syscalls.dirtyc0w,cve.cve-2016-5195
 This test case was failing on mem_write kernel call where
 CONFIG_SECURITY_CHROMIUMOS_READONLY_PROC_SELF_MEM return -EACCES

Change-Id: I7eb7641af4bf4cc0abaa0b75a002e5ea6a0d6ecd
Tracked-On: OAM-93086
Signed-off-by: rramakax <rohitx.kant@intel.com>
---
 groups/kernel/gmin64/config-lts/lts2019-chromium/x86_64_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/lts2019-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2019-chromium/x86_64_defconfig
index 4022dc5..594ff84 100644
--- a/groups/kernel/gmin64/config-lts/lts2019-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2019-chromium/x86_64_defconfig
@@ -6268,7 +6268,7 @@ CONFIG_SECURITY_SELINUX_CHECKREQPROT_VALUE=1
 # CONFIG_SECURITY_SAFESETID is not set
 # CONFIG_SECURITY_LOCKDOWN_LSM is not set
 # CONFIG_SECURITY_CHROMIUMOS is not set
-CONFIG_SECURITY_CHROMIUMOS_READONLY_PROC_SELF_MEM=y
+# CONFIG_SECURITY_CHROMIUMOS_READONLY_PROC_SELF_MEM is not set
 # CONFIG_INTEGRITY is not set
 CONFIG_DEFAULT_SECURITY_SELINUX=y
 # CONFIG_DEFAULT_SECURITY_DAC is not set
--
1.9.1

